dashboard/app/controllers/pd/application/facilitator_application_controller.rb

```diff
@@ -6,8 +6,10 @@ class FacilitatorApplicationController < ApplicationController
 
     def new
       # Block on production until we're ready to release and publicize the url
-      # TODO: Andrew - remove this line when we want to go live
-      return head :not_found if Rails.env.production?
+      # TODO: Andrew - remove this after we go live
+      if Rails.env.production? && !Gatekeeper.allows('pd_facilitator_application')
```

Comment:
This will prevent us from workshop admins from viewing the application early - suggest adding a conditional.

Also if you are feeling like it - it would be nice if Gatekeeper had a `disallows` method that effectively was returning `!Gatekeeper.allows`. But whatever